torchbenchmark/models/fastNLP/__init__.py

```diff
@@ -11,6 +10,11 @@
 import pickle
 import os
 
+if torchtext.__version__ >= "0.8.0":
+    from torchtext.datasets import AG_NEWS
+else:
+    from torchtext.experimental.datasets import AG_NEWS
+
```

Comment:
Rather than hardcoding string comparison, why not just try to import it from different location
```suggestion
try:
    from torchtext.datasets import AG_NEWS
except ImportError:
    # In older version of torchtext AG_NEWS is inside experimental module
    from torchtext.experimental.datasets import AG_NEWS

```